smart-contracts/contracts-common/concordium-contracts-common/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased changes
 
+- Fix `Display` implementation on `Timestamp` such that errors isn't returned but instead it falls back to an `u64` string.
```

Comment:
```suggestion
- Fix `Display` implementation on `Timestamp` such that it does not return an error if the timestamp is not representable in RFC3339 format, but instead it falls back to formatting as the number of milliseconds since the unix epoch.
```